ethereum/p2p/src/main/java/org/hyperledger/besu/ethereum/p2p/discovery/PeerDiscoveryAgent.java

```diff
@@ -314,6 +315,15 @@ protected void handleOutgoingPacket(final DiscoveryPeer peer, final Packet packe
                 if (err instanceof SocketException && err.getMessage().contains("unreachable")) {
                   LOG.debug(
                       "Peer {} is unreachable, packet: {}", peer, wrapBuffer(packet.encode()), err);
+                } else if (err instanceof UnsupportedAddressTypeException) {
+                  LOG.warn(
+                      "Unsupported address type exception when connecting to peer {}, this is likely due to ipv6 not being enabled at runtime. Enable debug trace to see full stacktrace",
```

Comment:
Should clarify last sentence to read "Set logging level to TRACE to see full stacktrace", because as written it is ambiguous which level will produce the better diagnostics.

Reply:
Yeah I think I was conscious about the length of the explanation, but that would make it lot easier to understand, will add it in!